cypress.json

```diff
@@ -1,3 +1,4 @@
 {
-  "baseUrl": "http://localhost:4200"
+  "baseUrl": "https://stage-revalidation.tis.nhs.uk",
```

Comment:
unsure how this will run on pipeline when hasn't been deployed?

Reply:
was looking at `https://www.cypress.io/blog/2019/11/20/drastically-simplify-your-testing-with-cypress-github-action/` a while ago? unsure if we may need this

Reply:
i see your point, i suppose this would mean the feature would have to be deployed first and the e2e tests come after. but not sure if this restriction is a good idea but not sure how to get around the cypress multiple domain restriction...

Reply:
thats a great article! I've added it to the pipeline ticket TISNEW-4706 (currently in the backlog at the moment). be worthwhile discussing the reval e2e pipeline with the other FEDS

Reply:
on git unbutu server npm start will launch it locally to the docker-image then cypress will run against it and pass or fail. All prior to deploying to staging